### DISCUSSION TITLE
Welcome to rust-miniscript Discussions!

### DISCUSSION THREAD

**Discussion by sanket1729** [rust-bitcoin/rust-miniscript#330] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by apoelstra** [comment]
I am Andrew Poelstra. With Sanket and @sipa I developed the Miniscript language; I started this library as a way to experiment with the different kinds of analysis on Bitcoin Script that I hoped Miniscript would enable. More recently I have been looking at extending Miniscript to work on top of richer scripting languages with [elements-miniscript](https://github.com/ElementsProject/elements-miniscript) and [rust-simplicity](https://github.com/ElementsProject/rust-simplicity).
